envs/env-lsdc-server-2022-2.0-py37.yml

```diff
@@ -6,3 +6,24 @@ dependencies:
   - conda-pack
   - pip:
     - ipython
+  - amostra <1,>=0.2.3
```

Comment:
Does it work as expected if the conda packages are listed after the pip ones? I normally add the pip dependencies to the end to avoid the potential confusion by conda.

Reply:
I see - I didn't know how significant the ordering is, but it makes sense now.